develop/BT-49 : BoongTam-RN develop 글꼴 및 색상 추가

ilchan123 opened the pull request:
[브랜치명](https://algoflow.atlassian.net/browse/KAN-58?atlOrigin=eyJpIjoiMjRhODNjNjg5N2IzNDcwY2EyZDNkMzFkYWIzMjJmYWMiLCJwIjoiaiJ9)

1. 내용요약: 글꼴 및 색상 수정 + 폰트 설정 추가 

2. 상세설명: 지정 글꼴과 폰트 사이즈를 정리했습니다. 

3. 팀원들에게 한마디: 화이팅입니다.

ilchan123 committed:
글꼴 및 색상 적용

ilchan123 committed:
글꼴 및 색상 적용

ilchan123 committed:
feat: 글꼴 및 색상

leeJongMin58 commented on constants/Colors.ts:
```diff
@@ -1,26 +1,34 @@
+  gray: {
+      gray500: '#212121',     // 진한 회색
+      gray400: '#616161',  
+      gray300: '#9F9F9F',  
```
gray300은 9E9E9E 아닌가요?

ilchan123 replied:
수정완료했습니다

leeJongMin58 commented on constants/Colors.ts:
```diff
@@ -1,26 +1,34 @@
+      orange100: '#FFB74D',
+      orange000: '#FFCC80',   // 연한 주황색
+  },
+  system: { 
```
red100, blue100, brown100 으로 이름 통일해주세여

ilchan123 replied:
수정완료했습니다

leeJongMin58 commented on constants/Typography.ts:
```diff
@@ -0,0 +1,106 @@
+            fontFamily: 'Roboto Mono',
+            fontSize: 57,
+            fontWeight: '500',
+            lineHeight: 150,
```
LineHeight이 150이 아니라 150%에요 전체 수정 부탁드려요

ilchan123 replied:
이게 타입 에러라고 떠서 % 적용이 안됩니다. 이걸 전해드린다는 게 까먹었습니다.
정확힌
[150%와 같은 퍼센트 표현은 React Native의 lineHeight에서 지원되지 않습니다.]입니다.

leeJongMin58 replied:
fontsize * 1.5 로 하시면 적용 됩니다

예를 들어 이런식으로요
fontFamily: 'Roboto Mono',
fontSize: 57,
fontWeight: '500',
lineHeight: 57 * 1.5

leeJongMin58 commented on constants/Typography.ts:
```diff
@@ -0,0 +1,106 @@
+            lineHeight: 150,
+            color: Color.gray.gray500,
+        },
+        heading_small_bold: {
```
볼드 적용도 안되어 있네용

ilchan123 replied:
bold를 하려고 했지만, 문자로 bold를 따로 지정할 경우 숫자랑 다르게, 별도의 설치 과정이 필요해서 숫자로 적용했습니다.
bold 타입과 normal에 대한 수치가 정의되어 있어서 700이 bold라고 보시면 됩니다.
아 저 부분은 700이 아니라 500이라고 되어있네요. 수정완료했습니다

leeJongMin58 replied:
설치 하시면 됩니다 ㅎㅎ

leeJongMin58 reviewed (commented):
나머지는 중간중간 리뷰 댓글로 남겨뒀습니다.
글씨체 폰트 다운 받아서 적용해주세요

일단 요정도

leeJongMin58 commented on app/(tabs)/index.tsx:
```diff
@@ -1,74 +1,214 @@
-}
+};
 
 const styles = StyleSheet.create({
```
앞으로 constants에 있는 color와 Typography를 사용 하기 위해서는 이러한 StyleSheet가 모든 뷰에 필요한거죠??

ilchan123 replied:
넵. 
Typo같은 여러 형식이 지정된 글꼴의 경우.
const styles = StyleSheet.create을 사용해야 [...] 스프레드 연산자로 적용해서 사용할 수 있습니다.

leeJongMin58 replied:
그러면 export로 다로 빼서 사용할까요?

leeJongMin58 commented on constants/Colors.ts:
```diff
@@ -1,26 +1,34 @@
-/**
```
확장자를 ts말고 js로 만들어 주셔도 됩니다. 아마 똑같이 돌아갈걸요?

ilchan123 replied:
바꿔서 적용하겠습니다

ilchan123 replied:
config에 영향을 주어 변경을 멈췄습니다.

leeJongMin58 replied:
방법을 찾아볼게요

ilchan123 committed:
feat: 글꽃 및 폰트 설정 추가

ilchan123 committed:
feat: index.tsx에서 색상글꼴 쓰는 법

ilchan123 reviewed (commented):
내용요약: index.tsx의 코드 사용 수정.

상세설명: 본문과 스타일이 함께 쓰이는 걸 따로 따로 사용할 수 있게 구분해 두었습니다. 

팀원들에게 한마디: 사용해 보시고 문제가 있으시면 알려주세요.

ilchan123 committed:
feat: 글꼴 수정

내용:
1. headig -> heading 으로 오타 수정

2. label 크기 명칭 수정(크기에 따른 분류)
기존: large, small, very_small
변경: large normal small

3. Typography 명칭 간략화
기존: Typography.heading.heading_large
변경: Typography.heading.large

남길말: 변경사항으로 명령어를 입력해주시면 됩니다.

ilchan123 committed:
feat: 글꼴 수정

내용:
1. headig -> heading 으로 오타 수정

2. label 크기 명칭 수정(크기에 따른 분류)
기존: large, small, very_small
변경: large normal small

3. Typography 명칭 간략화
기존: Typography.heading.heading_large
변경: Typography.heading.large

남길말: 변경사항으로 명령어를 입력해주시면 됩니다.

ilchan123 committed:
feat: 글꼴 수정

내용:
1. headig -> heading 으로 오타 수정

2. label 크기 명칭 수정(크기에 따른 분류)
기존: large, small, very_small
변경: large normal small

3. Typography 명칭 간략화
기존: Typography.heading.heading_large
변경: Typography.heading.large

남길말: 변경사항으로 명령어를 입력해주시면 됩니다.
